WordPress/src/main/java/org/wordpress/android/viewmodel/mlp/ModalLayoutPickerViewModel.kt

```diff
@@ -34,28 +46,62 @@ class ModalLayoutPickerViewModel @Inject constructor(
     private val _uiState: MutableLiveData<UiState> = MutableLiveData()
     val uiState: LiveData<UiState> = _uiState
 
+    private val contentUiState: ContentUiState
```

Comment:
Could you please elaborate a bit why you decided to introduce this non-nullable field? AFAIU _uiState already holds the current state - which might not be ContentUiState -> won't this (contentUiState) field be out-of-sync when for example `uiState` is `LoadingUiState`? It'll technically hold an empty/invalid state, right?

Reply:
I tried to avoid repeating the cast/check every time I wanted the `ContentUiState`. Thinking about this again making it non-nullable was not a good ideal and may cause more problems than it solves in the future.
Fixed with d968dd97eb86d7d85e7dc8069c84d45bccfe9d4e